tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "strictNullChecks": true,
     "resolveJsonModule": true,
     "noUnusedLocals": true,
-    "noImplicitAny": true
+    "noImplicitAny": true,
+    "noEmit": true
```

Comment:
This change seems unrelated. What does it do?

Reply:
I run `tsc` to see types error, `noEmit` needed to don't create `.js` files and see only errors. This added in another PR.